cxplat/cxplat_test/cxplat_memory_test.cpp

```diff
@@ -85,4 +86,18 @@ TEST_CASE("cxplat_duplicate_utf8_string", "[memory]")
     REQUIRE(memcmp(destination.value, source.value, source.length) == 0);
 
     cxplat_free_utf8_string(&destination);
+}
+
+TEST_CASE("cxplat_allocate_lookaside_list", "[memory]")
+{
+    cxplat_lookaside_list_t lookaside = {0};
```

Comment:
```suggestion
    cxplat_lookaside_list_t list= {0};
```
lookaside here is the adjective not the noun, list is the noun in the type name